Solnet.Serum/ISerumClient.cs

```diff
@@ -12,11 +13,21 @@ namespace Solnet.Serum
     /// </summary>
     public interface ISerumClient
     {
+        /// <summary>
+        /// The websocket connection state.
+        /// </summary>
+        WebSocketState State { get; }

```

Comment:
Is there any way to be notified of a disconnection from the underlying socket? The subscription object returned from the Subscribe* methods doesn't seem to have any events or other mechanism